airflow/dag_processing/bundles/git.py

```diff
@@ -40,20 +41,36 @@ class GitDagBundle(BaseDagBundle):
     :param repo_url: URL of the git repository
     :param tracking_ref: Branch or tag for this DAG bundle
     :param subdir: Subdirectory within the repository where the DAGs are stored (Optional)
+    :param ssh_conn_id: Connection ID for SSH connection to the repository (Optional)
     """
 
     supports_versioning = True
 
-    def __init__(self, *, repo_url: str, tracking_ref: str, subdir: str | None = None, **kwargs) -> None:
+    def __init__(
+        self,
+        *,
+        repo_url: str,
+        tracking_ref: str,
+        subdir: str | None = None,
+        ssh_conn_kwargs: dict[str, str] | None = None,
```

Comment:
```suggestion
        ssh_hook_kwargs: dict[str, str] | None = None,
```

These are really kwargs for `SSHHook`.  docstring is wrong too.